asm/non_matchings/menus/func_800B3554.s

```diff
@@ -166,7 +166,7 @@ glabel func_800B3554
 /* 0B43B0 800B37B0 24700005 */   addiu $s0, $v1, 5
 /* 0B43B4 800B37B4 0010CE00 */  sll   $t9, $s0, 0x18
 /* 0B43B8 800B37B8 00192603 */  sra   $a0, $t9, 0x18
-/* 0B43BC 800B37BC 0C02CD3A */  jal   func_800B34E8
+/* 0B43BC 800B37BC 0C02CD3A */  jal   open_css_grid_spot
```

Comment:
css?

Reply:
haha whoops the smash is sinking in. Character Select Screen

Reply:
How about something along the lines of `open_context_menu`? Prefer not to have acronyms as someone reading the code-base won't know the acronyms. Long names are always okay.

Reply:
oh sure. the function is used to check if there is an open spot to move the cursor on the choose your character screen. Returns FALSE if there is another cursor at the requested spot, TRUE otherwise. So, maybe something like `is_character_spot_free`?